src/D2L.Bmx/OktaAuthenticator.cs

```diff
@@ -95,22 +91,19 @@ bool nonInteractive
 		throw new BmxException( "Authentication Failed" );
 	}
 
-	private async Task<OktaAuthenticatedState> AuthenticateFromCacheAsync(
+	private async Task<bool> TryAuthenticateFromCacheAsync(
 		string org,
 		string user,
 		IOktaApi oktaApi
 	) {
 		string? sessionId = GetCachedOktaSessionId( user, org );
 		if( string.IsNullOrEmpty( sessionId ) ) {
-			return new( SuccessfulAuthentication: false, OktaSessionId: "" );
+			return false;
 		}
 
 		oktaApi.AddSession( sessionId );
-		string? userId = await oktaApi.GetMeResponseAsync( sessionId );
-		if( !string.IsNullOrEmpty( userId ) ) {
-			return new( SuccessfulAuthentication: true, OktaSessionId: userId );
-		}
```

Comment:
assigning user id to session id doesn't make sense... but `OktaSessionId` isn't used by any caller anyway, so removing it is the best.